FirstProject/assets/login.less

```diff
@@ -0,0 +1,62 @@
+//Login.less
+
+body {
+  background-color:#fafafa;
+}
+
+header {
```

Comment:
![header](https://user-images.githubusercontent.com/31060543/29551319-d5c567f4-871a-11e7-9834-9cc7dee58d1c.png)


1. Should I remove nesting from all .less files?
2. Is the class name good?